cli/install/cni-template.go

```diff
@@ -118,25 +120,10 @@ spec:
     metadata:
       labels:
         k8s-app: linkerd-cni
-      annotations:
```

Comment:
As a matter of convention, we typically add the `CreatedByAnnotation` on both the pod template as well as the parent object:
https://github.com/linkerd/linkerd2/blob/66070c26f4c5639fd94f99345b2cfba066b9134e/chart/templates/base.yaml#L151-L160

Reply:
👍